src/components/application_manager/src/application_impl.cc

```diff
@@ -661,25 +670,37 @@ void ApplicationImpl::WakeUpStreaming(
     {  // reduce the range of video_streaming_suspended_lock_
       sync_primitives::AutoLock lock(video_streaming_suspended_lock_);
       if (video_streaming_suspended_) {
-        application_manager_.OnAppStreaming(app_id(), service_type, true);
+        application_manager_.OnAppStreaming(
+            app_id(), service_type, StreamingState::kStarted);
         application_manager_.ProcessOnDataStreamingNotification(
             service_type, app_id(), true);
         video_streaming_suspended_ = false;
       }
     }
+
+    if (timer_len != 0) {
+      SDL_LOG_DEBUG("New estimated video playback time is: " << timer_len);
```

Comment:
@AKalinich-Luxoft 
Please change order comparing:
0 == timer_len

Reply:
@SNiukalov fixed in ce5bce1528d56f60e65492288d1f6d1c1f1b9372